src/com/sap/piper/ConfigurationLoader.groovy

```diff
@@ -1,44 +1,104 @@
 package com.sap.piper
 
+// script is present in the signatures in order to keep api compatibility.
+// The script referenced is not used inside the method bodies.
+
 @API(deprecated = true)
 class ConfigurationLoader implements Serializable {
+
+    static Map stepConfiguration(String stepName) {
+        return stepConfiguration(null, stepName)
```

Comment:
I think we should not call a deprecated method. Rather the other way around.

Reply:
Yes, thats better. I agree.